vault/resource_identity_oidc_assignment_test.go

```diff
@@ -0,0 +1,67 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccIdentityOIDCAssignment(t *testing.T) {
+	name := acctest.RandomWithPrefix("test-scope")
+	resourceName := "vault_identity_oidc_assignment.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testutil.TestAccPreCheck(t) },
+		Providers:    testProviders,
+		CheckDestroy: testAccCheckIdentityEntityDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIdentityOIDCAssignmentConfig_basic(name),
```

Comment:
Might be a good idea to add a test where `goup_ids` and `entity_ids` are empty.

Reply:
Updated with new empty case test